docs/start/qiskit-versioning.mdx

```diff
@@ -0,0 +1,126 @@
+---
```

Comment:
This will need to go into the `start/_toc.json`. I'm not sure where it makes sense to go, though. cc @javabster

Reply:
I added it the end of the toc here: https://github.com/Qiskit/documentation/pull/366/commits/2a48e8ee0a769dd242319f53cee13d67fd72a058 we can move it wherever makes sense though. I wasn't sure where to put it either.